integrations/milvus-document-store.md

```diff
@@ -33,60 +34,97 @@ toc: true
 ---
 
 ### Installation
-```console
-pip install -U milvus-haystack
+
+```shell
+pip install --upgrade pymilvus milvus-haystack
 ```
 
 ### Usage
 
-First, to start up a Milvus service, follow the ['Start Milvus'](https://milvus.io/docs/install_standalone-docker.md#Start-Milvus) instructions in the documentation. 
+By default, if you install the latest version of pymilvus, you don't need to start the milvus service manually.
```

Comment:
Can you give information about the latest version number here? Is it 2.4+?

Reply:
Yes, it is ,  2.4+ is ok,  2.4.3+ is more stable